### ISSUE TITLE
redesign the menu & some usability tweaks

### ISSUE THREAD

**Issue by VanDerLars** [SovereignCloudStack/website#94]
This project seems to more or less public related, so I would like to suggest some improvements.

**I suggest the following improvements for the menu/header:**
- Add menu point naming, and don't rely on the tool-tip of the links. 
  - Tool-tips are not visible on mobile devices. 
  - They are only shown if you hold your mouse still for a while, which is a bad UX.
  - They cannot not be read out by text-to-voice programs, for instance for disabled humans.
- Behind the `tel-href` is no telephone number. Only in the tool-tip is the number visible.
- Separate internal links from external Links visually. Maybe open external links in `_blank`. Users usually want to stay on one website until they change to foreign linked sites eventually.
- On desktops, the logo seems to big, on mobile the logo is to small 😄 

**Here are some ideas for the rest of the page:**
- Make the website more accessible for disabled people.
- Add the usage of the Cloudflare CDN to the `Datenschutzerklärung`/`data privacy statement` (important IMHO).

Cheers 🍺

**Comment by garloff** [comment]
Good suggestions -- would have been easier to split into several issues, so we can address them one-by-one.
First addressed issue with PR #95.
I am providing the css and fonts now locally, so no need to access cloudflare.
I think this is cleaner than changing the data protection statement.

**Comment by garloff** [comment]
To the menu discussion:
* All sub pages are also linked from the main text on the main page -- the toolbar at the top are just additional ways to navigate.
* I can replace the symbols with text or even have both ...

**Comment by garloff** [comment]
Navbar now has text in addition to icons.
External links now use target="_blank".
Telephone link in navbar has been removed.

**Comment by VanDerLars** [comment]
Nice!

> I am providing the css and fonts now locally, so no need to access cloudflare.
> I think this is cleaner than changing the data protection statement.

Best idea!

----


> Navbar now has text in addition to icons.
> External links now use target="_blank".
> Telephone link in navbar has been removed.

Much better 👍 

----

Last thing is then the mobile view. The menu is now very large on smartphones. Could be solved withoud JS over a little "hack", for example:

https://codepen.io/VanDerLars/pen/NWjpazj

```HTML
<div>
  <input id="cmn-toggle-7" class="hidden" type="checkbox" >
  <label for="cmn-toggle-7" class="clicker" tabindex="1">Click me</label>
  <div class="hiddendiv"></div>
</div>
```

```CSS
.clicker {
  width: 100px;
  height: 100px;
  background-color: blue;
  outline: none;
  cursor: pointer;
  color:#FFF;
  display:block;
}

.hiddendiv {
  display: none;
  height: 200px;
  background-color: green;
}

.clicker:focus+.hiddendiv {
  display: block;
}
.hidden{
  margin-left:-99999px;
}
input#cmn-toggle-7:checked + label + div{
 display:block;
}
```

**Comment by itrich** [comment]
> Last thing is then the mobile view. The menu is now very large on smartphones. Could be solved withoud JS over a little "hack", for example:

Hi @VanDerLars,

thank you for all your ideas and hints to improve the SCS website. The last of your suggestions is addressed separately with issue #103.
